package.json

```diff
@@ -75,7 +75,8 @@
     "angular-tree-component": "^6.0.0 || ^7.0.0",
     "c3": "^0.4.15",
     "ng2-dragula": "^1.5.0",
-    "ngx-bootstrap": "^1.8.0 || ^2.0.0"
+    "ngx-bootstrap": "^1.8.0 || ^2.0.0",
+    "rxjs-compat": "^6.1.0"
```

Comment:
Considering it's the application that performs the transpile (patternfly-ng doesn't use rxjs-compat), I'm thinking it should be the application that includes rxjs-compat. That way, an unused package is not installed by default for existing Angular 4 and 5 applications.

Since there are breaking changes with rxjs 6 (the reason for rxjs-compat), and the numerous challenges noted in #356, I'd like to create a new major version of patternfly-ng built with Angular 6 instead. Then, we can update to the latest versions of ngx-bootstrap 3.0.0, ngx-datatable 13.0.0, and webpack 4.

Note that patternfly-ng currently builds with Angular 4 in order to be compatible with 4 and 5. However, we cannot upgrade to ngx-bootstrap 3.0.0 and ngx-datatable 13.0.0 because they are built with Angular 6. In order to upgrade and be truly compatible, patternfly-ng must also build with Angular 6, which will be a breaking change.

Reply:
See: https://github.com/patternfly/patternfly-ng/pull/359